server/conn.go

```diff
@@ -513,47 +513,56 @@ func (cc *clientConn) writeReq(filePath string) error {
 	return errors.Trace(cc.flush())
 }
 
+var defaultLoadDataBatchCnt = 200000
+
+func batchInsertData(prevData, curData []byte, loadDataInfo *executor.LoadDataInfo) ([]byte, error) {
+	var err error
+	var reachLimit bool
+	for {
+		prevData, reachLimit, err = loadDataInfo.InsertData(prevData, curData)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		if !reachLimit {
+			break
+		}
+		if err = loadDataInfo.Ctx.Txn().Commit(); err != nil {
+			return nil, errors.Trace(err)
+		}
+		loadDataInfo.Ctx.NewTxn()
```

Comment:
error is not checked.
And we can call `NewTxn` only, it will commit current transaction.